projects/demo/src/pages/kit/date-range/date-range-mask-doc.component.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from '@maskito/kit';
 import {TuiDocExample} from '@taiga-ui/addon-doc';
 import {CHAR_EN_DASH, CHAR_NO_BREAK_SPACE, tuiPure} from '@taiga-ui/cdk';
+import {DEFAULT_DATE_RANGE_SEPARATOR} from 'projects/kit/src/lib/masks/date-range';
```

Comment:
Let's don't extend public API without real necessity (until somebody actually needs it).

Let' drop export of `DEFAULT_DATE_RANGE_SEPARATOR` from `@maskito/kit` and set `rangeSeparator` here as just ` – `.

Reply:
fixed